aries_cloudagent/config/default_context.py

```diff
@@ -127,6 +129,13 @@ async def load_plugins(self, context: InjectionContext):
         plugin_registry.register_plugin("aries_cloudagent.revocation")
         plugin_registry.register_plugin("aries_cloudagent.resolver")
         plugin_registry.register_plugin("aries_cloudagent.wallet")
+        plugin_registry.register_plugin("aries_cloudagent.anoncreds.anoncreds")
+        plugin_registry.register_plugin("aries_cloudagent.anoncreds.did_indy_registry")
+        plugin_registry.register_plugin("aries_cloudagent.anoncreds.did_web_registry")
+
+        # TODO: add anoncreds_plugins to argparse settings
+        for plugin_path in self.settings.get("anoncreds_plugins", []):
+            plugin_registry.register_plugin(plugin_path)
```

Comment:
Loading the anoncreds plugins will look the same as loading other plugins so we can reuse the `--plugin` flag instead of creating a new one